ui/package.json

```diff
@@ -47,6 +47,7 @@
         "coveralls": "^2.13.0",
         "css-loader": "^0.28.4",
         "emotion": "^7.2.2",
+        "emotion-server": "^7.3.2",
```

Comment:
Isn't this a dependency of `emotion`?

Reply:
Even if: we can't require a sub-dependency of another module, but be explicit about it :)

Reply:
Ah, I understand now, thanks for the clarification!

Reply:
oooh this may be a big help to me – nice one